solr/core/src/java/org/apache/solr/cloud/ExclusiveSliceProperty.java

```diff
@@ -74,8 +74,8 @@
   ExclusiveSliceProperty(ClusterState clusterState, ZkNodeProps message) {
     this.clusterState = clusterState;
     String tmp = message.getStr(ZkStateReader.PROPERTY_PROP);
-    if (StringUtils.startsWith(tmp, OverseerCollectionMessageHandler.COLL_PROP_PREFIX) == false) {
-      tmp = OverseerCollectionMessageHandler.COLL_PROP_PREFIX + tmp;
+    if (StringUtils.startsWith(tmp, CollectionAdminParams.PROPERTY_PREFIX) == false) {
```

Comment:
Minor: use `!` rather than `== false` (I know it's old code but you touched it :)

Reply:
Heh, you touched it - you own it :)